Realm/Realm/Helpers/Operator.cs

```diff
@@ -397,7 +397,7 @@ internal static class Operator
                  */
                 if (value is null)
                 {
-                    return Convert<RealmValue, TResult>(RealmValue.Null());
+                    return Convert<RealmValue, TResult>(RealmValue.Null);
```

Comment:
This should also be changed in Operator.tt, otherwise the next time the code is regenerated, it will revert to using Null as a method.